clients/unshaded/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -265,7 +269,11 @@ public synchronized void close() throws IOException {
    * @throws IOException if the underlying master RPC fails
    */
   synchronized void completeFile(long fid) throws IOException {
-    mFSMasterClient.completeFile(fid);
+    try {
+      mFSMasterClient.completeFile(fid);
+    } catch (TException e) {
```

Comment:
Can you also explain throwing IOExceptions in this file? It seems inconsistent with the PR message. So it'll be better to explain it more.